src/vaadin-grid-scroller.html

```diff
@@ -270,6 +270,8 @@
         const oldOffset = this._vidxOffset;
         if (this._accessIronListAPI(() => this._maxScrollTop) && this._virtualCount < this._effectiveSize) {
           this._adjustVirtualIndexOffset(delta);
+        } else {
+            this._vidxOffset = 0;
```

Comment:
Please change the indentation here, should be two spaces.

Reply:
Should be ok now.